mne/connectivity/spectral.py

```diff
@@ -442,29 +442,32 @@ def _epoch_spectral_connectivity(data, sig_idx, tmin_idx, tmax_idx, sfreq,
 
             for method in con_methods:
                 method.accumulate(con_idx, csd)
-    else:
-        # cwt_morlet mode
-        for i in range(0, n_cons, block_size):
+    elif mode in ('cwt_morlet',):  # reminder to add alternative TFR methods
+        for i_block, i in enumerate(range(0, n_cons, block_size)):
             con_idx = slice(i, i + block_size)
+            # XXX checkout conversion to slices here / index tricks
```

Comment:
some note from the past for a future version of myself who is as puzzled as you about its meaning. will remove it.